docs/src/main/sphinx/client/cli.rst

```diff
@@ -285,7 +285,21 @@ and prompts the CLI for your password:
 
 .. code-block:: text
 
-  ./trino --server https://trino.example.com --user=myusername --password
+  ./trino --server https://trino.example.com --user=exampleusername --password
```

Comment:
Maybe make the username more like an email address since that was one of the points of confusing, like exampleusername@example.com

Reply:
Usernames do NOT look like email in Trino typcially.